.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,36 @@
+name: Documentation bug
+description: Raise an issue related to content.
+title: "🐞 Documentation Bug or documentation feature: "
+labels: "documentation, to sort"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        * Please check that there is not an existing open [issue](https://github.com/tracim/tracim/labels/documentation) before you create a new one.
+  - type: input
+    attributes:
+      label: What content on github.com/tracim/tracim] is affected?
```

Comment:
Either missing a `[` or a `]` is forgotten here

Reply:
Fixed